### PR TITLE
CDS links post

### PR DESCRIPTION
**Pull request by rachelandrew**
Fixes: #6667 

Here's the post, also let me know if I missed anything! 

I want to add the recap video before publishing, it goes live at 11am PT so I've left that as a todo.

cc @mihajlija

### CODE REVIEW

File: src/site/content/en/blog/cds2021-updates/index.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+## Make the web more interoperable 
+
+Web compatibility is one of the biggest challenges for web developers, 
+and so Google has been working with other browser vendors to fix the top five compatibility pain points:  
+[flexbox](/learn/css/flexbox/), 
+[grid](/learn/css/grid/), 
+position: sticky, 
+[aspect-ratio](/aspect-ratio/), 
+and [transforms](/learn/css/transforms/).
```
Comment (mihajlija) [inline src/site/content/en/blog/cds2021-updates/index.md:28]: This link doesn't work, there's this though `learn/css/transitions/#transform`

File: src/site/content/en/blog/cds2021-updates/index.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+This post rounds up all the key announcements from the 2021 Chrome Dev Summit, with the links you need to find out more.
+
+## Make the web more interoperable 
+
+Web compatibility is one of the biggest challenges for web developers, 
+and so Google has been working with other browser vendors to fix the top five compatibility pain points:  
+[flexbox](/learn/css/flexbox/), 
+[grid](/learn/css/grid/), 
+position: sticky, 
+[aspect-ratio](/aspect-ratio/),
```
Comment (mihajlija) [inline src/site/content/en/blog/cds2021-updates/index.md:27]:
```suggestion
[`aspect-ratio`](/aspect-ratio/), 
```

File: src/site/content/en/blog/cds2021-updates/index.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+{% Aside %}
+The Microsoft Edge team took on the work of rewriting CSS grid layout as part of the Compat 2021 effort. 
+You can read about their work in the post 
+[Compat2021: Improving CSS Grid compatibility with GridNG](https://blogs.windows.com/msedgedev/2021/08/10/compat2021-css-grid-gridng/).
+{% endAside %}
+
+## Allow new classes of applications to run on the web
+
+[Project Fugu](/fugu-status/) is an effort that aims to allow new classes of applications to the web. 
+You can keep up to date with progress on the Fugu APIs at [fugu-tracker.web.app](https://fugu-tracker.web.app/)
```
Comment (mihajlija) [inline src/site/content/en/blog/cds2021-updates/index.md:42]:
```suggestion
You can keep up to date with progress on the Fugu APIs at [fugu-tracker.web.app](https://fugu-tracker.web.app/).
```

File: src/site/content/en/blog/cds2021-updates/index.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+Adobe recently announced that they are bringing Photoshop to the web, 
+a feat made possible by collaboration between Adobe, Google, and web standards organizations. 
+Find out more in the post [Photoshop's Journey to the web](/ps-on-the-web/).
```
Comment (mihajlija) [inline src/site/content/en/blog/cds2021-updates/index.md:46]:
```suggestion
Find out more in the post [Photoshop's journey to the web](/ps-on-the-web/).
```

File: src/site/content/en/blog/cds2021-updates/index.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+## Ensure privacy
+
+It's difficult for developers to meet the growing need for privacy when many applications were developed using third-party cookies and solutions that weren't designed with privacy in mind.
```
Comment (mihajlija) [inline src/site/content/en/blog/cds2021-updates/index.md:50]:
```suggestion
Many common web functions rely on third-party cookies and other cross-site tracking mechanisms that weren’t designed with privacy in mind, making it difficult for developers to meet the growing need for privacy.
```

File: src/site/content/en/blog/cds2021-updates/index.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+## Help you to create beautiful and responsive sites
+
+The [new responsive design](/new-responsive/) is so much more than just screen—or container—size. 
+You can learn how to create modern responsive sites in our course 
+[Learn Design](/learn/design/). 
+The first five modules launch today, the rest will be posted over the coming weeks.
+
+New CSS properties have landed in Chrome and other browsers this year to make creating beautiful experiences easier. 
+Find out more about [accent-color](/accent-color/) and  [size-adjust](/css-size-adjust/).
```
Comment (mihajlija) [inline src/site/content/en/blog/cds2021-updates/index.md:100]:
```suggestion
Find out more about [`accent-color`](/accent-color/) and [`size-adjust`](/css-size-adjust/).
```

File: src/site/content/en/blog/cds2021-updates/index.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+This post rounds up all the key announcements from the 2021 Chrome Dev Summit, with the links you need to find out more.
+
+## Make the web more interoperable 
+
+Web compatibility is one of the biggest challenges for web developers, 
+and so Google has been working with other browser vendors to fix the top five compatibility pain points:  
+[flexbox](/learn/css/flexbox/), 
+[grid](/learn/css/grid/), 
+position: sticky,
```
Comment (mihajlija) [inline src/site/content/en/blog/cds2021-updates/index.md:26]:
```suggestion
`position: sticky`, 
```

**Review by mihajlija — APPROVED** [review]

File: src/site/content/en/blog/cds2021-updates/index.md (hunk 1 above)
Comment (rachelandrew) [reply src/site/content/en/blog/cds2021-updates/index.md:28]:
```suggestion
and [transforms](/learn/css/transitions/#transform).
```

**Pull request merged**